source/extensions/filters/http/ext_proc/processor_state.cc

```diff
@@ -20,19 +20,26 @@ using envoy::service::ext_proc::v3::TrailersResponse;
 
 void ProcessorState::onStartProcessorCall(Event::TimerCb cb, std::chrono::milliseconds timeout,
                                           CallbackState callback_state) {
+  ENVOY_LOG(debug, "Start external processing call");
   callback_state_ = callback_state;
-  if (!message_timer_) {
-    message_timer_ = filter_callbacks_->dispatcher().createTimer(cb);
+
+  // Skip starting timer For FULL_DUPLEX_STREAMED body mode.
```

Comment:
Without timer, how we are going to handle timeout situation that side stream server doesn't respond? Do you handle it somewhere in this PR?

Reply:
If the side stream server does not respond,  the router filter idle timeout will kick in, and destroy ext_proc filter.  This will be same as a backend server does not respond to a client request.